keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/callbacks/SlackCallbackHandler.kt

```diff
@@ -0,0 +1,89 @@
+package com.netflix.spinnaker.keel.slack.callbacks
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.netflix.spinnaker.keel.notifications.NotificationType.MANUAL_JUDGMENT
+import com.netflix.spinnaker.keel.serialization.configuredObjectMapper
+import com.netflix.spinnaker.kork.web.exceptions.InvalidRequestException
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.net.URLDecoder
+
+@Component
+class SlackCallbackHandler(
+  private val manualJudgmentCallbackHandler: ManualJudgmentCallbackHandler
+) {
+  companion object {
+    val mapper: ObjectMapper = configuredObjectMapper()
+  }
+
+  private val log by lazy { LoggerFactory.getLogger(javaClass) }
+
+  fun handleCallback(request: String?) {
+    if (request == null) {
+      log.error("the request body coming from slack callback is null.")
+      //throw exception?
```

Comment:
Why are you making the parameter nullable in the first place?

Reply:
do you think it's better to check it in the controller?